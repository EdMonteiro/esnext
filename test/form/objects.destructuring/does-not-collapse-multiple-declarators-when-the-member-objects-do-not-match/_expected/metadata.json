{
  "objects.destructuring": [
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "a",
          "typeAnnotation": null
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "object": {
            "type": "Identifier",
            "name": "foo",
            "typeAnnotation": null
          },
          "property": {
            "type": "Identifier",
            "name": "a",
            "typeAnnotation": null
          },
          "computed": false
        }
      ]
    },
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "b",
          "typeAnnotation": null
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "object": {
            "type": "Identifier",
            "name": "bar",
            "typeAnnotation": null
          },
          "property": {
            "type": "Identifier",
            "name": "b",
            "typeAnnotation": null
          },
          "computed": false
        }
      ]
    }
  ]
}