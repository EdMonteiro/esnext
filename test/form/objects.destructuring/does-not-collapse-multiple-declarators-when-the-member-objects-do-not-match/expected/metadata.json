{
  "objects.destructuring": [
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "a"
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "computed": false,
          "object": {
            "type": "Identifier",
            "name": "foo"
          },
          "property": {
            "type": "Identifier",
            "name": "a"
          }
        }
      ]
    },
    {
      "ids": [
        {
          "type": "Identifier",
          "name": "b"
        }
      ],
      "inits": [
        {
          "type": "MemberExpression",
          "computed": false,
          "object": {
            "type": "Identifier",
            "name": "bar"
          },
          "property": {
            "type": "Identifier",
            "name": "b"
          }
        }
      ]
    }
  ]
}